addons/auth_signup/data/mail_template_data.xml

```diff
@@ -341,5 +341,65 @@
             <field name="auto_delete" eval="True"/>
         </record>
 
+        <!-- Email template for new device login alert -->
+        <record id="alert_new_device_email" model="mail.template">
+            <field name="name">Settings: Alert New Device</field>
+            <field name="model_id" ref="base.model_res_users"/>
+            <field name="subject">New Connection to your Account</field>
+            <field name="email_from">"{{ object.company_id.name }}" &lt;{{ (object.company_id.email or user.email) }}&gt;</field>
```

Comment:
You could use email_formatted field available on res.company model. Same for user, fallback on email_formatted.